source/includes/common/bucket-replication.rst

```diff
@@ -0,0 +1,167 @@
+.. start-create-replication-remote-targets-cli
+
+Use the :mc-cmd:`mc admin bucket remote add` command to create a replication target from each deployment to the appropriate bucket on the destination deployment. 
+MinIO supports *one* remote target for an origin bucket to destination bucket combination. 
+
```

Comment:
This reads a little awkward to me. 

What we're trying to get across is that a bucket can have N remote targets, but those targets must each be *unique*. A bucket cannot have two remote target configs that resolve to the same remote host/bucket combination. 

I think.

Reply:
Taking a stab at updated phrasing to better capture the need here.